app/src/main/res/layout/activity_help_legend.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
HAHAHAHAHHA HILARIOUS JUST KIDDING. MY SAVING OF STRING IS WHAT MADE HELP LEGEND CRASH HAHAHAHAHAHAHAHAHAHAHA